lib/tests/streamlit/elements/plotly_chart_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import plotly.express as px
+from parameterized import parameterized
+
+import streamlit as st
+from streamlit.errors import StreamlitAPIException
+from tests.delta_generator_test_case import DeltaGeneratorTestCase
+
+
+class PyDeckTest(DeltaGeneratorTestCase):
+    def test_basic(self):
+        """Test that plotly object works."""
+        df = px.data.gapminder().query("country=='Canada'")
+        fig = px.line(df, x="year", y="lifeExp", title="Life expectancy in Canada")
+        st.plotly_chart(fig)
+
+        el = self.get_delta_from_queue().new_element
+        print(el)
+        self.assertTrue(el.plotly_chart.figure.spec != None)
+        self.assertTrue(el.plotly_chart.figure.config != None)
+
+    @parameterized.expand(
+        [
+            ("streamlit", "streamlit"),
+            (None, ""),
+        ]
+    )
+    def test_theme(self, theme_value, proto_value):
+        """Test that plotly object works."""
+        df = px.data.gapminder().query("country=='Canada'")
+        fig = px.line(df, x="year", y="lifeExp", title="Life expectancy in Canada")
+        st.plotly_chart(fig, theme=theme_value)
+
+        el = self.get_delta_from_queue().new_element
+        self.assertEqual(el.plotly_chart.theme, proto_value)
+
+    def test_bad_theme(self):
+        """Test that plotly object works."""
+        df = px.data.gapminder().query("country=='Canada'")
+        fig = px.line(df, x="year", y="lifeExp", title="Life expectancy in Canada")
+        with self.assertRaises(StreamlitAPIException) as exc:
+            st.plotly_chart(fig, theme="bad_theme")
+
+        self.assertEqual(
+            f"""You set theme=\"bad_theme\" while Streamlit charts only support theme=”streamlit” or theme=None to fallback to the default library theme. """,
```

Comment:
Same as the comment above, you don't need to use the multiline docstring notion (via `"""`) here